sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -126,6 +129,25 @@ def tags(self):
         # type: () -> List[str]
         return self._tags
 
+    @property
+    def fully_qualified_reference(self):
+        # type: () -> str
+        return "{}/{}{}{}".format(
+            _host_only(self._registry),
+            self._repository_name,
+            ":" if _is_tag(self._digest) else "@",
+            _strip_alg(self._digest)
```

Comment:
Just to make sure I understand correctly -- `self._digest` would also be holding the tag if `_is_tag()` returns true?

Reply:
Yes that is correct